eng/common/templates/steps/source-build.yml

```diff
@@ -63,11 +63,6 @@ steps:
       targetRidArgs='/p:TargetRid=${{ parameters.platform.targetRID }}'
     fi
 
-    runtimeOsArgs=
-    if [ '${{ parameters.platform.runtimeOS }}' != '' ]; then
-      runtimeOsArgs='/p:RuntimeOS=${{ parameters.platform.runtimeOS }}'
-    fi
```

Comment:
we should also remove the usage of `runtimeOsArgs` on line 83, though these changes are in eng/common so they'd need to be done in the arcade repo.